src/Pages/Home.js

```diff
@@ -0,0 +1,41 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { useNavigate } from 'react-router-dom';
+import { displayStock, stockDetails } from '../redux/financialStatements/finStatements';
+
+const Home = () => {
+  const stocks = useSelector((state) => state.stocks);
+  const dispatch = useDispatch();
+  const navigate = useNavigate();
+  useEffect(() => {
+    dispatch(displayStock());
+  }, []);
+
+  const stockClicked = (stock) => {
+    dispatch(stockDetails(stock));
+  };
+
+  return (
+    <>
+      <div><h3>Cryptocurrency Trading Data</h3></div>
+      <div className="row justify-content-center">
+        {stocks.map((stock) => (
+          <div className="col-sm-4 colDiv" key={stock.symbol}>
+            <button
+              type="button"
+              onClick={() => {
+                stockClicked(stock);
+                navigate('/details');
+              }}
+            >
+              {stock.symbol}
+
+            </button>
+          </div>
+        ))}
+      </div>
+    </>
+  );
+};
```

Comment:
- [ ] Great job 👏🏻  however according to the project requirements, The Home page shout has a list of Cryptocurrencies that could be filtered using given parameters, e.g. Cryptocurrency name. 👨🏻‍🎨  Kindly make sure to add a filter to your home page so that users can make a search on any kind of Cryptocurrency just by typing their name 😉 👍🏻

Reply:
can you explain further, there's no search bar provided in the design, where should i implement this

Reply:
Hi @kamzzy, It's one of the project requirements to filter on your home page so that users can filter your data from the API using  e.g. Cryptocurrency name hope that is clear now 👍🏻